dts/bindings/pinctrl/hpmicro,hpm-pinctrl.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 hpmicro
+# SPDX-License-Identifier: Apache-2.0
+
+description: HPMicro PinMux
```

Comment:
please extend this documentation with some more details, check other vendors as a reference.